admin_honeypot/models.py

```diff
@@ -3,6 +3,26 @@
 from admin_honeypot import listeners
 
 
+class LoginAttempt(models.Model):
+    username = models.CharField(_("username"), max_length=255, blank=True, null=True)
+    ip_address = models.GenericIPAddressField(_("ip address"), protocol='both', blank=True, null=True)
+    session_key = models.CharField(_("session key"), max_length=50, blank=True, null=True)
+    user_agent = models.TextField(_("user-agent"), blank=True, null=True)
+    timestamp = models.DateTimeField(_("timestamp"), auto_now_add=True)
+    path = models.TextField(_("path"), blank=True, null=True)
+
+    class Meta:
+        verbose_name = _("login attempt")
+        verbose_name_plural = _("login attempts")
+        ordering = ('timestamp',)
+
+    def __str__(self):
+        return self.username
+from django.db import models
+from django.utils.translation import gettext_lazy as _
```

Comment:
this import is ok, but just work for Django 4, if anyone is using Django 3, this is version is not working with it, I think is better import djanngo, and do an if like:

`
import django


if django.VERSION < (2,):
    from django.utils.translation import ugettext_lazy as _
else:
    from django.utils.translation import gettext_lazy as _
`